docs/api/taxons.rst

```diff
@@ -0,0 +1,858 @@
+                        "level": 2,
+                        "position": 0,
+                        "translations": [],
+                        "images": [],
+                        "_links": {
+                            "self": {
+                                "href": "\/api\/v1\/taxons\/1037"
+                            }
+                        }
+                    },
+                    {
+                        "id": 1038,
+                        "code": "women",
+                        "children": [],
+                        "left": 5,
+                        "right": 6,
+                        "level": 2,
+                        "position": 1,
+                        "translations": [],
+                        "images": [],
+                        "_links": {
+                            "self": {
+                                "href": "\/api\/v1\/taxons\/1038"
+                            }
+                        }
+                    }
+                ],
+                "left": 2,
+                "right": 7,
+                "level": 1,
+                "position": 0,
+                "translations": [],
+                "images": [],
+                "_links": {
+                    "self": {
+                        "href": "\/api\/v1\/taxons\/1036"
+                    }
+                }
+            }
+        ],
+        "left": 1,
+        "right": 8,
+        "level": 0,
+        "position": 0,
+        "translations": [],
+        "images": [],
+        "_links": {
+            "self": {
+                "href": "\/api\/v1\/taxons\/1035"
+            }
+        }
+    }
+
+Creating Taxon
+--------------
+
+To create new taxon you will need to call the ``/api/v1/taxons/`` endpoint with ``POST`` method.
+
+Definition
+^^^^^^^^^^
+
+.. code-block:: text
+
+    POST /api/v1/taxons/
+
++------------------------------------+----------------+--------------------------------------+
+| Parameter                          | Parameter type | Description                          |
++====================================+================+======================================+
+| Authorization                      | header         | Token received during authentication |
++------------------------------------+----------------+--------------------------------------+
+| code                               | request        | **(unique)** Taxon identifier        |
++------------------------------------+----------------+--------------------------------------+
+
+Example
+.......
+
+To create new taxon use the below method.
+
+.. code-block:: bash
+
+    $ curl http://demo.sylius.org/api/v1/taxons/ \
+        -H "Authorization: Bearer SampleToken" \
+        -H "Content-Type: application/json" \
+        -X POST \
+        --data '
+            {
+                "code": "toys"
+            }
+        '
+
+.. note::
+
+    If you want to create you taxon under other taxon you should pass also a parent code.
```

Comment:
`your taxon as a child of another taxon, you should pass also the parent taxon's code`